pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ManagedLedgerCompressionTest.java

```diff
@@ -34,7 +34,7 @@
  * ManagedLedgerInfo compression configuration test.
  */
 
-@Test(groups = {"broker", "broker-jdk8"})
+@Test(groups = {"broker", "broker-jdk17"})
```

Comment:
just remove it. The reason why this specific test was marked for broker-jdk8 was that there's a change between Java 8 and Java 11 that showed up in this test. It's now obsolete, so remove the broker-jdk8 group completely.

Reply:
removed.